Project2/Part_B/Card.cpp

```diff
@@ -15,6 +15,12 @@ Card::Card(CARDVALUE_T val, SUIT_T suit)
   this->next = (Card*) nullptr;
 }
 
+Card::Card(const Card& other) {
+  value = other.value;
+  suit = other.suit;
+  next = other.next; // TODO: ensure that we should copy the next var
```

Comment:
Are we sure next should be initialized this way, and not ass nullptr?

Reply:
That is what I was trying to figure out as well, but figured I would build it this way because it is easy to switch it to a nullptr.